cmd/webhook/main.go

```diff
@@ -103,26 +103,44 @@ func main() {
 var (
 	_ resourcesemantics.SubResourceLimited = (*crdNoStatusUpdatesOrDeletes)(nil)
 	_ resourcesemantics.VerbLimited        = (*crdNoStatusUpdatesOrDeletes)(nil)
+
+	_ resourcesemantics.SubResourceLimited = (*crdEphemeralContainersUpdatesOrDeletes)(nil)
+	_ resourcesemantics.VerbLimited        = (*crdEphemeralContainersUpdatesOrDeletes)(nil)
 )
 
 type crdNoStatusUpdatesOrDeletes struct {
 	resourcesemantics.GenericCRD
 }
 
+type crdEphemeralContainersUpdatesOrDeletes struct {
```

Comment:
Bit of a nit. I'd prefer this to be just:
crdEphemeralContainers

Previously it had the NoStatusUpdatesOrDeletes, and now that it only pertains to EphemeralContainers and it doesn't have the 'No' there it's a bit misleading cause I read it as it's only for Updates Or Deletes for EphemeralContainers :)

Reply:
👍🏻